<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>만든놈 : Epinephrine00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>이메일 : epi00nephrine@gmail.com</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>GitHub</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Discord</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
